src/model/submissions.model.ts

```diff
@@ -0,0 +1,111 @@
+import {
+    // JoinColumn, // TODO: Uncomment this with FK constraint once the course entity is merged
+    //ManyToOne, // TODO: Uncomment this with FK constraint once the course entity is merged
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn
+} from "typeorm";
+
+
+//import UserModel from './users.model' // TODO: Uncomment this with FK constraint once the course entity is merged
+
+@Entity('submissions')
+export default class SubmissionModel {
+    @PrimaryGeneratedColumn()
+    id: number      //submission id, change to submission id?
+
+    @CreateDateColumn({name: "created_at"})
+    createdAt: Date
+
+    @UpdateDateColumn({name: "updated_at"})
+    updatedAt: Date
+
+    @DeleteDateColumn({name: "deleted_at"})
+    deletedAt?: Date
+
+    @Column({name: "submission_id"})
+    submissionId: number
+
+
+
+    //Foreign key
+    // TODO: Uncomment this with FK constraint once the course entity is merged
+    // @JoinColumn({name: "course_id"})
+    // @ManyToOne(() => UserModel)
+    // courseId: number
+
+    // TODO: Update this with FK constraint once the course entity is merged
+    @Column({name: "course_id"})
+    courseId: number
+
+
+
+
+    //Foreign key
+    // TODO: Uncomment this with FK constraint once the course entity is merged
+    // @JoinColumn({name: "assignment_id"})
+    // @ManyToOne(() => UserModel)
+    // assignmentId: number
+
+    // TODO: Update this with FK constraint once the course entity is merged
+    @Column({name: "assignment_id"})
+    assignmentId: number
+
+
+
+    //Foreign key
+    // TODO: Uncomment this with FK constraint once the course entity is merged
+    // @JoinColumn({name: "user_id"})
+    // @ManyToOne(() => UserModel)
+    // userId: number
+
+    // TODO: Update this with FK constraint once the course entity is merged
+    @Column({name: "user_id"})
```

Comment:
```
  // Foreign key
  @JoinColumn({ name: 'user_id' })
  @ManyToOne(() => UserModel)
  userId: number
  ```

  This is how you add a forign key. Do this. Leave it commented out for `courseId`, and `assignmentId`